### DISCUSSION TITLE
Looking for mod plugin help droping expert bags on normal

### DISCUSSION THREAD

**Discussion by Frankcolo** [Pryaxis/TShock#2056] [Q&A]
I'm running a private server for me and a friend of mine and was wondering if there is a plugin to allow expert bags to drop off of normal difficulty bosses?

I tried creating my own but am apparently hopeless with c# any pointers (or link to working plugin) would be appreciated.

**Comment by Kojirremer** [comment]
In theory, you could use Loot Table Editor (assuming it will be updated for 1.4) for that, but at least on its current state, it interacts with monsters dropping multiple items on death (like its drops, plus several healing hearts) in a funny way, replacing all drops with your added loot, so I wouldn't recommend editing a boss' loot table with it until someone actually updates it properly, which will most likely not happen, as it hasn't been updated (other than API ticks) in several years.
